### PR TITLE
ethernet/ethertype: remove ng_ prefix

### PR DESCRIPTION
**Pull request by authmillenon**
I used

```bash
git mv sys/include/net/ng_ethertype.h sys/include/net/ethertype.h
git mv sys/include/net/ng_ethernet.h sys/include/net/ethernet.h
git mv sys/include/net/ng_ethernet/ sys/include/net/ethernet/
git grep --name-only -i 'ng_ether' | xargs sed -i -e 's/ng_\(ether\)\([^ ]\+\)\( \+[/(].*\)/\1\2   \3/gI' -e 's/ng_\(ether\)/\1/gI'
```

to convert

### CODE REVIEW

File: sys/include/net/ethernet.h [hunk 1]
```diff
@@ -19,35 +19,35 @@
...
+#define ETHERNET_FCS_LEN        (4)     /**< number of bytes in the FCS
                                          *  (frame check sequence) */
 
 /**
  * @brief maximum number of bytes in an ethernet frame (without FCS)
  */
-#define NG_ETHERNET_FRAME_LEN   (NG_ETHERNET_DATA_LEN + \
-                                 sizeof(ng_ethernet_hdr_t))
-#define NG_ETHERNET_MIN_LEN     (64)    /**< minimum number of bytes in an
+#define ETHERNET_FRAME_LEN      (ETHERNET_DATA_LEN + \
```
Comment (OlegHahm) [inline sys/include/net/ethernet.h]: should fit in one 80 char line now

File: sys/include/net/ethernet.h [hunk 2]
```diff
@@ -19,35 +19,35 @@
...
+                                 sizeof(ethernet_hdr_t))
+#define ETHERNET_MIN_LEN        (64)    /**< minimum number of bytes in an
                                          * ethernet frame (with FCF) */
 
 /**
  * @brief maximum number of bytes in an ethernet frame (with FCF)
  */
-#define NG_ETHERNET_MAX_LEN     (NG_ETHERNET_FRAME_LEN + \
-                                 NG_ETHERNET_FCS_LEN)
+#define ETHERNET_MAX_LEN        (ETHERNET_FRAME_LEN + \
```
Comment (OlegHahm) [inline sys/include/net/ethernet.h]: here too

**Comment by authmillenon** [comment]
Rebased and addressed comments

**Comment by OlegHahm** [comment]
ACK! Squash, wait for Travis, and go!

**Comment by authmillenon** [comment]
Squashed.

**Pull request merged**